src/components/BottomNav/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { AppBar, BottomNavigation, BottomNavigationAction, createStyles, makeStyles } from '@material-ui/core';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import LocalFloristIcon from '@material-ui/icons/LocalFlorist';
+import StorefrontIcon from '@material-ui/icons/Storefront';
+import EmojiNatureIcon from '@material-ui/icons/EmojiNature';
+import { useHistory } from 'react-router-dom';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    appBar: {
+      top: 'auto',
+      bottom: 0,
+    }}));
+
+export const BottomNav = () => {
+  const [currentPage, setCurrentPage] = useState("/user/myGardens");
+  const classes = useStyles();
+  // TODO: decide how to change views on button click, fix issue with page changing
+  let history = useHistory();
+  const handlePageChange = (_event: React.ChangeEvent<{}>, newValue: string) => {
+    setCurrentPage(newValue);
+    // history.push(newValue)
```

Comment:
Currently having issue where using history. push here requires a double tap for bottom nav to transition to new view. Not sure why...

Reply:
Is this for every transition or on first load?